packages/react/src/primitives/Input/Input.tsx

```diff
@@ -39,6 +40,7 @@ const InputPrimitive: Primitive<InputProps, 'input'> = (
     classNameModifier(ComponentClassNames.Input, size),
     className
   );
+  const { isDisabled: isFieldsetDisabled } = useFieldset();
```

Comment:
This might be a silly thing, but should we just call this something different so we don't have to remap the name in every component?

Reply:
Has the same thought, we could just name the property `isFieldsetDisabled`

Reply:
yeah 🙃 that makes more sense lol, updating

Reply:
Updated in latest commit